doc/src/manual/faq.md

```diff
@@ -615,6 +615,9 @@ have, it ends up having a substantial cost due to compilers (LLVM and GCC) not g
 around the added overflow checks. If this improves in the future, we could consider defaulting
 to checked integer arithmetic in Julia, but for now, we have to live with the possibility of overflow.
 
+In the meantime, safe integer operations can be achieved trough the use of external libraries such as [safeInteger.jl](https://github.com/JeffreySarnoff/SaferIntegers.jl). Note that, as stated previously, the use of these libraries signically increases the execution time of Julia. You can follow the status of the discussion [here](https://github.com/JuliaLang/julia/issues/855).
```

Comment:
"significantly". But In my opinion it would be more correct to say that "arithmetic operations with safe numeric types are significantly slower" than talking about the execution time of Julia as the new types do not substitute native types.

Reply:
```suggestion
In the meantime, safe integer operations can be achieved trough the use of external libraries such as [SaferIntegers.jl](https://github.com/JeffreySarnoff/SaferIntegers.jl). Note that, as stated previously, the use of these libraries signically increases the execution time of Julia. You can follow the status of the discussion [here](https://github.com/JuliaLang/julia/issues/855).
```